src/commands/moderators/deleteAllMessage.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  ApplicationCommandType,
+  ChannelType,
+  DiscordAPIError,
+  GuildMember,
+  PermissionsBitField,
+} from 'discord.js'
+
+import { CommandHandlerConfig } from '../../types/CommandHandlerConfig.js'
+
+export default {
+  data: {
+    name: 'Delete All',
```

Comment:
Should we name it more explicitly?

```suggestion
    name: 'Delete all User's messages',
```

Reply:
it's too long for context menu

<img width="416" alt="image" src="https://github.com/creatorsgarten/kaogeek-discord-bot/assets/70970263/18e4fd4f-be1f-41dd-8c6a-dd73e81f09cd">

Reply:
What's about "Prune messages", should be explicit enough for moderator.